src/assets/social-media-icons/facebook.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 80 80" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
+    <g transform="matrix(1,0,0,1,-1.42109e-14,90.0476)">
+        <g id="Facebook" transform="matrix(0.078125,0,0,0.078125,40,-50.0476)">
+            <g transform="matrix(1,0,0,1,-512,-512)">
+                <g>
```

Comment:
Is the multiple grouping are all necessary?

fyi - https://www.w3.org/TR/SVG2/struct.html#GElement

Reply:
I took care of avoiding the most <g> tag nesting as possible, but there is still a need for <g> tags containing a transform attribute to be nested. I also cleaned up the unnecessary id attributes.